auditor.js

```diff
@@ -15,12 +15,18 @@
  */
 'use strict';
 
-module.exports = function(audits) {
-  return function audit(results) {
-    var flattenedAudits = results.reduce(function(prev, curr) {
+class Auditor {
+
+  _flattenArtifacts(artifacts) {
```

Comment:
ha, I didn't mean to state that so strongly. More like they can be static since they aren't keeping track of state or anything